### ISSUE TITLE
Compile error (error: 'mem_fn' is not a member of 'std')

### ISSUE THREAD

**Issue by shinchiro** [kcat/openal-soft#253]
Compile with current git head, 5a9a2cb

```
[1/11] cd /mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft && /usr/bin/cmake -D GIT_EXECUTABLE=/usr/sbin/git -D LIB_VERSION=1.19.1 -D SRC=/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/version.h.in -D DST=/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft-build/version.h -P /mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/version.cmake
[2/11] Building CXX object CMakeFiles/OpenAL.dir/Alc/effects/equalizer.cpp.obj
FAILED: CMakeFiles/OpenAL.dir/Alc/effects/equalizer.cpp.obj 
/mnt/c/base/build/install/bin/x86_64-w64-mingw32-g++  -DAL_ALEXT_PROTOTYPES -DAL_BUILD_LIBRARY -DAL_LIBTYPE_STATIC -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGE_FILES -D_WIN32 -D_WIN32_WINNT=0x0502 -I/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/include -I/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/common -I. -I/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/OpenAL32/Include -I/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/Alc -I/mnt/c/base/build/install/mingw/include -std=c++11  -O2 -g -D_DEBUG   -Winline -Wall -Wextra -fno-math-errno -MD -MT CMakeFiles/OpenAL.dir/Alc/effects/equalizer.cpp.obj -MF CMakeFiles/OpenAL.dir/Alc/effects/equalizer.cpp.obj.d -o CMakeFiles/OpenAL.dir/Alc/effects/equalizer.cpp.obj -c /mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/Alc/effects/equalizer.cpp
/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/Alc/effects/equalizer.cpp: In member function 'virtual ALboolean ALequalizerState::deviceUpdate(ALCdevice*)':
/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/Alc/effects/equalizer.cpp:104:28: error: 'mem_fn' is not a member of 'std'
                       std::mem_fn(&BiquadFilter::clear));
                            ^~~~~~
/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/Alc/effects/equalizer.cpp:104:28: note: 'std::mem_fn' is defined in header '<functional>'; did you forget to '#include <functional>'?
/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/Alc/effects/equalizer.cpp:33:1:
+#include <functional>
 #include "filters/defs.h"
/mnt/c/base/build/packages/openal-soft-prefix/src/openal-soft/Alc/effects/equalizer.cpp:104:28:
                       std::mem_fn(&BiquadFilter::clear));
                            ^~~~~~
```

**Comment by iSLC** [comment]
Does your compiler (_actually the standard library being used_) support c++11? And if so, is it enabled?

**Comment by DeadSix27** [comment]
@iSLC Including <functional> fixes building on my end, same error as above otherwise.

**Comment by kcat** [comment]
Fixed with commit fc8da0c16b34a964c637c721ff944e8e6a5f3277.
